.hooks/pre-commit

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+
+# case insensitivity for regex 
+shopt -s nocasematch
+
+# file types regexes
+INTEGRATION_REGEX="(integration-).*(.yml)"
+PLAYBOOK_REGEX="^(?!testPlaybooks).*(playbook-).*(.yml)"
```

Comment:
need to make sure we don't have `yaml` artifacts (scripts, integrations, playbooks)

Reply:
We don't have - validated